.github/workflows/refresh-dev.yml

```diff
@@ -25,8 +25,7 @@ jobs:
           proxy_port: ${{ secrets.SSH_PROXY_PORT }}
           script: |
             cd /srv/tae-dev && \
-            /usr/local/bin/docker-compose -f docker-compose.cloud.yml exec -T app ./artisan migrate:fresh --force && \
-            /usr/local/bin/docker-compose -f docker-compose.cloud.yml exec -T app ./artisan db:seed DevSeeder --force
+            /usr/local/bin/docker-compose -f docker-compose.cloud.yml exec -T app ./artisan db:refresh
```

Comment:
If we add the scheduler to the dev environment, could we run `artisan schedule:work` here instead?

Reply:
`artisan schedule:work` doesn't exit. 

I would probably launch a cron every minute like `/usr/local/bin/docker-compose -f docker-compose.cloud.yml exec -T app ./artisan schedule:run`.